tests/e2e/test_reborn_capability_policy_xyzorg.py

```
@@ -0,0 +1,820 @@
+    Each `projection_snapshot`/`projection_update` carries the full renderable
+    `ProductProjectionState`. We read until the run blocks on a gate or reaches a
+    terminal state, so a caller can tell whether a capability reached dispatch.
+    """
+    import json as _json
+
+    events_url = f"{API}/threads/{thread_id}/events?token={token}"
+    last_items: list = []
+    client_timeout = aiohttp.ClientTimeout(total=timeout + 2, sock_read=timeout + 2)
+    try:
+        async with aiohttp.ClientSession(timeout=client_timeout) as session:
+            async with session.get(events_url, headers={"Accept": "text/event-stream"}) as response:
```

Ruff (0.15.18)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)